Sources/FluentMySQL/MySQLDatabase.swift

```diff
@@ -33,6 +33,20 @@ public final class MySQLDatabase: Service {
         self.password = password
         self.database = database
     }
+    
+    /// Initialize MySQLDatabase with a DB URL
+    public convenience init?(databaseURL: String) {
+        guard let url = URL(string: databaseURL),
+            url.scheme == "mysql",
+            url.pathComponents.count >= 2,
+            let hostname = url.host,
+            let username = url.user,
+            let password = url.password
```

Comment:
Password should not be a hard requirement.

Reply:
Oh, yeah. Empty passwords can be `nil`

Reply:
Good catch!